### PR TITLE
fix(analytics): Fix SSL issue with analytics on frontend

### PR DESCRIPTION
**Pull request by dexter-mh-lee**
Fix SSL issue with analytics on frontend and allow adding index prefix to the analytics index. 
- Cleaned up old elasticsearch setup in the meantime

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml [hunk 1]
```diff
@@ -84,10 +90,26 @@ spec:
...
+                secretKeyRef:
+                  name: {{ $.Values.global.credentialsAndCertsSecrets.name }}
+                  key: {{ $envVarValue }}
+            {{- end }}
+            {{- end }}
             - name: ELASTIC_CLIENT_HOST
               value: "{{ .Values.global.elasticsearch.host }}"
             - name: ELASTIC_CLIENT_PORT
               value: "{{ .Values.global.elasticsearch.port }}"
+            {{- if .Values.global.datahub_analytics_enabled }}
```
Comment (jjoyce0510) [inline datahub-kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml:112]: Why is this located here as opposed to above?

File: gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "index_patterns": ["datahub_usage_event-*"],
+  "index_patterns": ["PREFIXdatahub_usage_event-*"],
```
Comment (jjoyce0510) [inline gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json:2]: PREFIX will get correctly populated here?

Comment (dexter-mh-lee) [reply gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json:2]: sed is used to do string replace. Couldn't think of a better way here. Any ideas?

File: datahub-kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml (hunk 1 above)
Comment (dexter-mh-lee) [reply datahub-kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml:112]: Good point. Done

File: gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json (hunk 2 above)
Comment (jjoyce0510) [reply gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json:2]: I see... hmm not really

**Review by jjoyce0510 — APPROVED** [review]
LGTM. Thanks

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**